lib/pbench/server/auth/auth.py

```diff
@@ -57,23 +58,41 @@ def get_current_user_id() -> Optional[str]:
     return str(user.id) if user else None
 
 
-def encode_auth_token(time_delta: datetime.timedelta, user_id: int) -> str:
+def encode_auth_token(time_delta: timedelta, user_id: int) -> Tuple[str, datetime]:
     """Generates an authorization token for an internal user ID.
 
     Args:
         time_delta : Token lifetime
         user_id : Authorized user's internal ID
 
     Returns:
-        JWT token string
+        JWT token string, expiration
```

Comment:
It would be clearer to state explicitly that the function returns a Tuple (and, then we can say what's in the tuple).